api/src/main/java/org/hawkular/inventory/api/model/MetricType.java

```diff
@@ -157,7 +155,7 @@ private Update() {
 
         public Update(Map<String, Object> properties, MetricUnit unit) {
             super(properties);
-            this.unit = unit;
+            this.unit = unit == null ? MetricUnit.NONE : unit;
```

Comment:
On 13 Jul 2015, at 17:53, Jirka Kremser wrote:
>
> agree, it's tricky. I can think of an example where we/feed/whoever 
> discover something and we don't know the units, so we set it to `NONE` 
> and user can manually set the units to enrich the inventory metadata

Yes. That would count as the first/initial setting.
But beyond that I believe that it does not necessarily make sense.

-- 
hwr@pilhuhn.de  m:0179/207 4919  b:http://pilhuhn.blogspot.com